internal/rbd/controllerserver.go

```diff
@@ -98,20 +98,23 @@ func (cs *ControllerServer) parseVolCreateRequest(
 	req *csi.CreateVolumeRequest) (*rbdVolume, error) {
 	// TODO (sbezverk) Last check for not exceeding total storage capacity
 
-	isMultiNode := false
+	isMultiWriter := false
 	isBlock := false
 	for _, capability := range req.VolumeCapabilities {
 		// RO modes need to be handled independently (ie right now even if access mode is RO, they'll be RW upon attach)
-		if capability.GetAccessMode().GetMode() == csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER {
-			isMultiNode = true
+		// below capability check indicates that we support both {SINGLE_NODE or MULTI_NODE} WRITERs and the `isMultiWriter`
```

Comment:
Might be useful to add a `isMultiWriter([]VolumeCapability)` function so that comments and long if-statements do not clutter this function so much. It will also make it easy to add a unit-test for the checks on the new capability without regressing on the previous behaviour.